src/main/java/org/openlmis/cce/web/VolumeController.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org.
+ */
+
+package org.openlmis.cce.web;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import org.openlmis.cce.domain.FunctionalStatus;
+import org.openlmis.cce.dto.VolumeDto;
+import org.openlmis.cce.repository.InventoryItemRepository;
+import org.slf4j.ext.XLogger;
+import org.slf4j.ext.XLoggerFactory;
+import org.slf4j.profiler.Profiler;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class VolumeController extends BaseController {
+
+  private static final XLogger XLOGGER = XLoggerFactory.getXLogger(VolumeController.class);
+
+  @Autowired
+  private InventoryItemRepository inventoryRepository;
+
+  /**
+   * Get all volume CCE Inventory items for specified facility.
+   *
+   * @return volume
+   */
+  @GetMapping(value = "/{facilityId}/volume")
```

Comment:
I am not sure if this is the correct mapping. If I think correctly, the full path will be /api/{facilityId}/volume. We probably need some base resource like inventoryItem, catalogItem, or even facility:

- /api/inventoryItems/{facilityId}/volume
- /api/catalogItems/{facilityId}/volume
- /api/facilities/{facilityId}/volume

Reply:
From the start, I was thinking of "/api/facilities/{facilityId}/volume" (I simply forgot to put "facilities" into the path in the ticket description) so I would like this option the most. Second option would be "/api/inventoryItems/volume" with 
required "facilityId" parameter since "/api/inventoryItems/{facilityId}/volume" is confusing - I imagine users put inventory item id instead of facility one.